webapp/indexfunctions.js

```diff
@@ -2897,14 +2904,30 @@ function showPropertiesWindow(graph, cell, diagRoot) {
         file_input.accept = '.au';
         file_input.style.display = 'none';
 
+        //Button to browse file in READC_f
+        var readc_file_btn = document.createElement("button");
+        readc_file_btn.innerHTML = 'Browse';
+        readc_file_btn.type = "button";
+        readc_file_btn.name = "readc_f_browser";
+        readc_file_btn.id = "readc_f_browser";
+        readc_file_btn.style.cssFloat = "right";
+        readc_file_btn.style.marginTop = "-5px";
+
+        //File browser to store file which we browse.
+        var file_input_c = document.createElement('input');
+        file_input_c.name = "file_readc_f";
+        file_input_c.type = 'file';
+        file_input_c.accept = '.c,.bin,.o,.exe';
```

Comment:
```suggestion
        file_input_c.accept = '.datas,.dat,.bin';
```
In writec.c, we are using .datas extension while writing.
Also, .c, .o, and .exe should not be allowed.